cloudformation/media-atom-maker.json

```diff
@@ -55,6 +55,10 @@
             "Description": "Name of the upload tracking dynamo table",
             "Type": "String"
         },
+        "ManualPlutoTable": {
```

Comment:
all our dynamo tables exist in a separate stack (not sure why). For this reason, we pass them in to the main template as params to craft the [policy](https://github.com/guardian/media-atom-maker/blob/master/cloudformation/media-atom-maker.json#L838). As we have to do this, it makes it simpler to `cat` to the conf file in the launch config so as not to duplicate table names...

I'd love to just merge the two stacks into one template...